encryption-at-rest.md

```diff
@@ -128,3 +128,9 @@ When restoring the backup, both `--s3.sse` and `--s3.sse-kms-key-id` should NOT
 ```
 ./br restore full --pd <pd-address> --storage "s3://<bucket>/<prefix> --s3.region <region>"
 ```
+
+## Encryption at rest for TiFlash <span class="version-mark">New in v4.0.5</span>
+
+TiFlash supports encryption at rest starting from v4.0.5. Data keys are generated by TiFlash. All files written by TiFlash (including TiFlash Proxy), including data files, schema files, temporary files, etc., are encrypted by the current data key. The encryption algorithms, encryption configuration (configuration items in the `tiflash-learner.toml` file) of TiFlash, and the meanings of monitoring items are consistent with those of TiKV.
```

Comment:
```suggestion
TiFlash supports encryption at rest since v4.0.5. Data keys are generated by TiFlash. All files (including data files, schema files, and temporary files) written into TiFlash (including TiFlash Proxy) are encrypted using the current data key. The encryption algorithms, the encryption configuration (in the `tiflash-learner.toml` file) supported by TiFlash, and the meanings of monitoring metrics are consistent with those of TiKV.
```